content/hi/docs/reference/glossary/cgroup.md

```diff
@@ -0,0 +1,17 @@
+---
+title: सीग्रुप (cgroup)
+id: cgroup
+date: 2019-06-25
+full_link:
+short_description: >
+  Linux प्रक्रियाओं का एक समूह जो चुनिंदा रूप से संसाधनों को अलग, प्रबंधित और प्रतिबंधित करता है।

```

Comment:
```suggestion
  Linux प्रक्रियाओं (processes) का एक समूह जो चुनिंदा रूप से संसाधनों को अलग, प्रबंधित और प्रतिबंधित करता है।
```
I think having processes written in brakets makes this easier to understand for some people?